src/System.Diagnostics.Process/tests/ProcessStartInfoTests.cs

```diff
@@ -365,11 +365,12 @@ public void TestUserCredentialsPropertiesOnWindows()
             p.StartInfo.LoadUserProfile = true;
             p.StartInfo.UserName = username;
             p.StartInfo.PasswordInClearText = password;
+            bool hasStarted = false;
 
             SafeProcessHandle handle = null;
             try
             {
-                p.Start();
+                hasStarted = p.Start();
```

Comment:
The above statements aren't expected to throw any exceptions.. If so failing test is better no..